packages/strapi-plugin-upload/config/functions/bootstrap.js

```diff
@@ -1,15 +1,17 @@
 'use strict';
 /**
- * Upload plugin bootstrapi.
+ * Upload plugin bootstrap.
  *
  * It initializes the provider and sets the default settings in db.
  */
 
 module.exports = async () => {
+  const name = 'upload';
```

Comment:
Can we name it `pluginName` for readibility ?

Reply:
You can also get the name from the package.json

```
const pluginPkg = require('../../package.json');

const pluginId = pluginPkg.name.replace(/^strapi-plugin-/i, '');

export default pluginId;
```